### DISCUSSION TITLE
How to access current camera image and save it?

### DISCUSSION THREAD

**Discussion by cyberhenoch** [sceneview/sceneform-android#161] [Q&A]
There is a good way to save image from arSceneView Surface [here](https://github.com/ThomasGorisse/sceneform-android-sdk/discussions/134). But is it possible to get a surface with raw camera feed without the rendered scene? I would need a resolution greater than 640x480 from the ARCore frame.acquireCameraImage()

**Comment by ThomasGorisse** [comment]
I think it will be complicated since AR Core locks other camera usage.
I even wasn't able to simply enable the camera flash light since CameraManager will not let you play with him while AR Core is.

You can have a look at what AR Core shared camera is proposing but I don't think it's made for that.

Can you ask it on the [AR Core repo](https://github.com/google-ar/arcore-android-sdk) and link it here?

**Reply by cyberhenoch** [reply]
Thomas, thanks. In fact I managed to implement shared camera basing on the official example. However it slows down the processing and I was unable to make a single shot, just continuous camera feed. There were some problems with creating and releasing the ImageReader too.
I will reproduce it and post the question on SO as the ARCore repo info suggests.
